### PR TITLE
[13.0][MIG] hr_timesheet_sheet_autodraft: Migration to 13.0

### PR DESCRIPTION
**Pull request by davejames**

### CODE REVIEW

File: hr_timesheet_sheet_autodraft/__manifest__.py [hunk 1]
```diff
@@ -2,25 +2,18 @@
...
-    'installable': True,
-    'application': False,
-    'summary': (
-        'Automatically draft a Timesheet Sheet for every time entry that does'
-        ' not have a relevant Timesheet Sheet existing.'
+    "name": "HR Timesheet Sheet Auto-draft",
+    "version": "12.0.1.0.0",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/timesheet",
+    "author": "Brainbean Apps, " "Odoo Community Association (OCA)",
```
Comment (alexey-pelykh) [inline hr_timesheet_sheet_autodraft/__manifest__.py]: This `" "` after-effect of auto-format should be removed manually, but not anyhow important

**Review by alexey-pelykh — APPROVED** [review]

File: hr_timesheet_sheet_autodraft/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "license": "AGPL-3",
+    "installable": True,
+    "application": False,
+    "summary": (
+        "Automatically draft a Timesheet Sheet for every time entry that does"
+        " not have a relevant Timesheet Sheet existing."
+    ),
+    "depends": ["hr_timesheet_sheet",],
```
Comment (alexey-pelykh) [inline hr_timesheet_sheet_autodraft/__manifest__.py]: `,]` should be changed to `]` or linter will complain

**Comment by alexey-pelykh** [comment]
When running pre-commit, please ensure that there's no "Failed" ones

File: hr_timesheet_sheet_autodraft/__manifest__.py (hunk 1 above)
Comment (davejames) [reply hr_timesheet_sheet_autodraft/__manifest__.py]: I have removed the extra `" "`

File: hr_timesheet_sheet_autodraft/__manifest__.py (hunk 2 above)
Comment (davejames) [reply hr_timesheet_sheet_autodraft/__manifest__.py]: I have removed the trailing comma here and in all other places

**Comment by davejames** [comment]
pre-commit no longer has any failures

**Comment by alexey-pelykh** [comment]
Unfortunately, travis still fails

**Comment by davejames** [comment]
> Unfortunately, travis still fails

it seems to be failing on the hr_timesheet_sheet module though which is not touched by my commits

`hr_timesheet_sheet/migrations/13.0.1.0.0/post-migration.py:4: [W7936(missing-manifest-dependency), ] Missing external dependency "openupgradelib" from manifest. More info: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#external-dependencies
`

**Comment by alexey-pelykh** [comment]
Going to fix that in https://github.com/OCA/timesheet/pull/346

**Comment by alexey-pelykh** [comment]
please rebase on updated 13.0 branch

**Review by alexey-pelykh — APPROVED** [review]

**Comment by davejames** [comment]
I'm not sure why this is still failing, I rebased and your changes to '.isort.cfg' are present but travis is still failing on hr_timesheet_sheet

**Comment by alexey-pelykh** [comment]
Unfortunately, that goes a bit beyond my knowledge, not sure what's that about. @pedrobaeza maybe you know?

**Comment by alexey-pelykh** [comment]
Please rebase one more time

**Comment by davejames** [comment]
rebased, checks are passing now

**Comment by alexey-pelykh** [comment]
@astirpe @MiquelRForgeFlow if possible, please take a look

**Comment by davejames** [comment]
@astirpe @pedrobaeza any chance of a review on this one?

**Comment by alexey-pelykh** [comment]
for crying out loud, we'll never get proper pace of migration with current velocity...

**Pull request merged**